elliptic-curve/src/point.rs

```diff
@@ -40,9 +40,9 @@ pub trait AffineCoordinates {
 
 /// Normalize point(s) in projective representation by converting them to their affine ones.
 #[cfg(feature = "arithmetic")]
-pub trait BatchNormalize<Points: ?Sized>: group::Curve {
+pub trait BatchNormalize<Points: ?Sized> {
     /// The output of the batch normalization; a container of affine points.
-    type Output: AsRef<[Self::AffineRepr]>;
+    type Output;
```

Comment:
Seems like these are breaking changes that might need a corresponding PR to https://github.com/RustCrypto/elliptic-curves

Reply:
Oh sorry, missed that